# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../at76c503a/linux-2.6.16.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- at76c503a/at76c503-fw_skel.c.vanilla	2006-05-18 10:46:37.772394500 +0200
+++ at76c503a/at76c503-fw_skel.c	2006-05-18 10:46:57.637636000 +0200
@@ -89,10 +89,10 @@
 /* structure for registering this driver with the usb subsystem */
 
 static struct usb_driver module_usb = {
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,4,20)
+	.driver = {
 	owner:      THIS_MODULE,
-#endif
 	name:	    DRIVER_NAME,
+	},
 	probe:	    at76c50x_probe,
 	disconnect: at76c50x_disconnect,
 	id_table:   dev_table,
